libs/sdk-backend-tiger/src/fixtures/referenceObjects.ts

```diff
@@ -0,0 +1,25 @@
+// (C) 2022 GoodData Corporation
+import { ObjRef } from "@gooddata/sdk-model";
+
+export const stageStatuses = [
+    { title: "Lost", uri: "/api/elements/referenceworkspace/obj/1100/elements?id=460496" },
```

Comment:
The uri values are strange, on tiger they are often the same as title, i.e. they almost never look like uris.